2.4/root/usr/bin/run-mongod

```diff
@@ -47,9 +47,7 @@ function cleanup() {
   fi
   echo "=> Shutting down MongoDB server ..."
   if [ -f "${MONGODB_PID_FILE}" ]; then
-    kill -2 $(cat ${MONGODB_PID_FILE})
-  else
-    pkill -2 mongod
+    mongod --shutdown $mongo_common_args
```

Comment:
https://github.com/openshift/mongodb/commit/d7efd424116c7fe087094f8887940eb8e96ac1c6

@mfojtik Do you remember why you've added this form of `cleanup()`